pkg/container/container.go

```diff
@@ -140,8 +142,35 @@ func (c *container) SpecAddAnnotations(sb *sandbox.Sandbox, containerVolumes []o
 
 	// Preserve the sandbox annotations. OCI hooks may re-use the sandbox
 	// annotation values to apply them to the container later on.
+	// The sandbox annotations are already filtered for the allowed
+	// annotations, there is no need to check it additionally here.
 	for k, v := range sb.Annotations() {
-		c.spec.AddAnnotation(k, v)
+		if strings.HasPrefix(k, pkgAnnotations.OCISeccompBPFHookAnnotation) {
+			// The OCI seccomp BPF hook
+			// (https://github.com/containers/oci-seccomp-bpf-hook)
+			// uses the annotation io.containers.trace-syscall as indicator
```

Comment:
nit of a nit, only change if you've other changes
```suggestion
			// uses the annotation io.containers.trace-syscall as an indicator
```

Reply:
Fixed as suggested, had to rebase.